src/frontend/js/components/LtiConsumer/index.tsx

```diff
@@ -19,7 +21,15 @@ const LtiConsumer = ({ id }: LtiConsumerProps) => {
   useAsyncEffect(async () => {
     await fetch(`/api/v1.0/plugins/lti-consumer/${id}/context/`)
       .then(checkResponseStatus)
-      .then(setContext)
+      .then((ltiContext: LtiConsumerContext) =>
+        setContext({
+          ...ltiContext,
+          content_parameters: {
+            ...ltiContext.content_parameters,
+            ...(user && { lis_person_sourcedid: user.username }),
```

Comment:
should we also send the email or not ?

Reply:
I don't have this information right now. Without using a custom route to retrieve user information as we will do with fonzie, currently the only one solution is to make two request to OpenEdX:
- first request `/me` to retrieve the username of the logged in user
- then a second request to retrieve account information from this `username`

Reply:
Ok the username is enough for now.